api/src/main/java/org/openmrs/module/registrationcore/RegistrationCoreConstants.java

```diff
@@ -14,29 +14,49 @@
 package org.openmrs.module.registrationcore;
 
 public final class RegistrationCoreConstants {
-	
+
 	/**
 	 * Specifies the identifier source to use when generating patient identifiers
 	 */
 	public static final String GP_IDENTIFIER_SOURCE_ID = "registrationcore.identifierSourceId";
-	
+
 	public static final String DATE_FORMAT_STRING = "yyyy-MM-dd HH:mm:ss";
-	
-	public static final String TOPIC_NAME = "org.openmrs.module.registrationcore.RegistrationEvent";
-	
+
+	public static final String REGISTRATION_EVENT_TOPIC_NAME = "org.openmrs.module.registrationcore.RegistrationEvent";
+
+	public static final String EDIT_EVENT_TOPIC_NAME = "org.openmrs.module.registrationcore.EditEvent";
```

Comment:
"edit event" isn't specific enough. Call it "patient edit event" or something like that.